lib/screens/wods/add_wod.dart

```diff
@@ -4,11 +4,14 @@ import 'package:flutter_form_builder/flutter_form_builder.dart';
 //my imports
 import 'package:kabod_app/core/presentation/constants.dart';
 import 'package:kabod_app/core/presentation/routes.dart';
+import 'package:kabod_app/screens/home/model/wod_model.dart';
+import 'package:kabod_app/screens/wods/components/delete_wod_button.dart';
 import 'package:kabod_app/screens/wods/components/add_wod_form.dart';
 
 class AddWodScreen extends StatefulWidget {
```

Comment:
Since this screen is now used to create, edit and delete a WoD, let's change its name to `WodEditorScreen` something similar please.